valhalla/odin/util.h

```diff
@@ -61,6 +61,15 @@ const locales_singleton_t& get_locales();
  */
 const std::unordered_map<std::string, std::string>& get_locales_json();
 
+/**
+ * Convert the input units, in either imperial or metric, into meters.
+ * @param   distance units to convert to meters
```

Comment:
should mention that this is either miles or kilometers? maybe a better name would be in order as well?

Reply:
That would depend on whether `is_metric` is true or not. A caller could call this function to convert km to meters or miles to meters. Do you want to change the name of `units` param to something else? (units seemed generic and thats what I see us using in `odin`). I''m open to suggestions. Let me know what you had in mind.

Reply:
I think I understood what you meant. The input param name should reflect that the units are in km or miles. Correct?

Reply:
yes i just m ean that neither the doc string nor the variable name tell you what units should be